tests/target/type-alias-where-clauses.rs

```diff
@@ -0,0 +1,20 @@
+type Foo
+where
+    A: B,
+    C: D,
+= E;
+
+type Foo
+where
+    A: B,
+    C: D,
+= E
+where
+    F: G,
+    H: I;
```

Comment:
This seems like an interesting case. Is it valid to have two where clauses?

Reply:
No, but it's not denied by the parser, so I think rustfmt shouldn't explode when we have it.

Reply:
I appreciate the clarification on this

Reply:
> so I think rustfmt shouldn't explode when we have it.

I'd like to adopt this as a new motto for us 🤣